modules/ROOT/partials/configuration/revisionhistory_fetch_revision.adoc

```diff
@@ -0,0 +1,87 @@
+[[revisionhistory_fetch_revision]]
+== `revisionhistory_fetch_revision`
+
+The plugin calls this function to fetch a revision's content when a revision is selected in the revision history dialog. This function is called with the revision object that was selected in the revision history dialog. The function should return a Promise that resolves with the revision content.
```

Comment:
```suggestion
The plugin uses this option to update for when either the selected and the closest revisions have no content. The function should return a Promise that resolves with an updated revision.
```

Reply:
My thoughts, the word `should` could potentially be confusing, to me it implies it `may` or `may not` return a Promise?

Reply:
[25b2863](https://github.com/tinymce/tinymce-docs/pull/3349/commits/25b28633e6cd8d91f5b5b4e4263a3622d194380c)